client/src/main/java/tds/assessment/Segment.java

```diff
@@ -25,13 +25,12 @@
     private List<Item> items;
     private Set<Strand> strands;
 
-    /**
-     * Empty constructor for frameworks
-     */
-    private Segment() {}
-
     public Segment(String key) {
         this.key = key;
+        languages = new ArrayList<>();
```

Comment:
If we have collections in the object I'd prefer them to be empty non null.  Makes it easier in calling code so you don't have to do `if(segment.getStrands != null) { for(segment.strands) }`